plugins/jdg72/pom.xml

```diff
@@ -15,6 +15,7 @@
 
    <properties>
       <version.infinispan>${jdg72.infinispan.version}</version.infinispan>
+      <version.infinispan.public>9.1.4.Final</version.infinispan.public>
```

Comment:
maybe `.upstream`

Reply:
I have been using `version.infinispan.public` since `jdg62` so even though that is a better name, I'll probably stick with this!

Reply:
ok